web-library/Api/BookReservation/Entity/Reservation.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace web_library.Api.BookReservation.Entity
+{
+    using System.ComponentModel.DataAnnotations;
+    using web_library.Api.Book.Entity;
+    [Table("reservations")]
+    public class Reservation
+    {
+        [Column("id")]
+        [Key]
+        public int Id { get; set; }
+        [Column("reservation_start_date")]
+        public DateOnly reservationStartDate { get; }
+        [Column("reservation_end_date")]
+        public DateOnly reservationEndDate { get; }
+        [Column("book_copy_id")]
+        public int bookCopyId { get; set; }
+        public BookCopy bookCopy { get; set; } = null!;
+
+        public Reservation()
+        {
+            reservationStartDate = new();
+            reservationEndDate = reservationStartDate.AddMonths(+1);
```

Comment:
rezerwacja trwa 3 dni jezeli nie jest wypozyczona a jak jest to 3 dni od oddania trzeba zrobic serwis ktory sprawdza czy nie upłynął przypadkiem czas rezerwacji na podstawie aktualnej daty